node/src/command.rs

```diff
@@ -125,7 +131,7 @@ impl SubstrateCli for Cli {
     }
 
     fn load_spec(&self, id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
-        load_spec(id, self.para_id.unwrap_or(1000).into())
+        load_spec(id, self.para_id, None, Some(vec![2000, 2001]), None)
```

Comment:
When is this used?

Reply:
On all other commands that are not build-spec. So launching a network `tanssi-node --dev` will mock the containers

Reply:
Ok and we were already doing that because the old `load_spec` always added those mock chains, except when the chain_id was a `path`, so same as we do now.

Reply:
That's right